src/pages/transactions/[id].js

```diff
@@ -0,0 +1,5 @@
+import Transaction from '../../components/Transaction';
+
+export default function TransactionDetails() {
+  return <Transaction />;
+}
```

Comment:
Please make the Transaction component the page and remove unncessary layer

Reply:
Done